dfget/core/api/supernode_api.go

```diff
@@ -53,6 +53,8 @@ type SupernodeAPI interface {
 	ReportPiece(node string, req *types.ReportPieceRequest) (resp *types.BaseResponse, e error)
 	ServiceDown(node string, taskID string, cid string) (resp *types.BaseResponse, e error)
 	ReportClientError(node string, req *types.ClientErrorRequest) (resp *types.BaseResponse, e error)
+	Get(url string, resp interface{}) error
```

Comment:
Why does make the `get` and `post` method to public? They are not the apis of supernode.